### DISCUSSION TITLE
Is it possible to automatically start a journey for manually imported users?

### DISCUSSION THREAD

**Discussion by vi1world** [parcelvoy/platform#468] [Q&A]
Hi,
I'm trying Parcelvoy after using many marketing automation tools.
One feature that I don't see is triggering a journey automatically whenever someone is manually added to the list.
Or is it in the 'Schedule' feature? If I set a journey to trigger everyday, will it re-triggered for users who are already in the journey? Is it possible to trigger immediately for people who are manually added into a list?

Thank you very much

**Comment by pushchris** [comment]
@vi1world there are a few ways to do what you are looking for, but in general we recommend always using events to trigger journeys unless you are wanting them to initialize in a recurring time-based fashion. There is a `user_created` event which is triggered every time a user is created (both manually and through events) which you can use to start journeys. You can then further filter by any criteria you would want

**Reply by vi1world** [reply]
Thank you.

Where can I find a list of available system events to use?
